vivarium/core/engine.py

```diff
@@ -711,7 +711,8 @@ def _invoke_process(
                 self.parallel[path] = ParallelProcess(
                     process, bool(self.profiler))
             # trigger the computation of the parallel process
-            self.parallel[path].update(interval, states)
+            self.parallel[path].send_command(
+                'next_update', (interval, states))
```

Comment:
this is very elegant! `send_command()` will greatly expand how we use parallel processes. great.